README.md

```diff
@@ -68,23 +68,38 @@ zuul:
     enabled: true 
     repository: REDIS 
     behind-proxy: true
-    default-policy: #optional - will apply unless specific policy exists
-      limit: 10 #optional - request number limit per refresh interval window
-      quota: 1000 #optional - request time limit per refresh interval window (in seconds)
-      refresh-interval: 60 #default value (in seconds)
-      type: #optional
-        - user
-        - origin
-        - url
     policies:
-      myServiceId:
```

Comment:
revert back to policies map, we want the api to change as little as possible to support this change, we can change each entry to contain a list of policies and keep the policy by service approach

Reply:
The original policy option is based entirely on service ids, but in many cases we want to be able to restrict the flow of the entire gateway system rather than a service module.